app/src/androidTest/java/com/duckduckgo/cookies/impl/SQLCookieRemoverTest.kt

```diff
@@ -74,15 +76,6 @@ class SQLCookieRemoverTest {
         assertTrue(success)
     }
 
-    @Test
-    fun whenNoCookiesStoredAndRemoveExecutedThenResultTrue() = runTest {
```

Comment:
Test is useless, in reality we never try to remove cookies if the are none in the higher call so we are not testing anything here.

Reply:
🔥